packages/client/contexts/AppContext.tsx

```diff
@@ -109,6 +110,9 @@ export const AppContextProvider: React.FC<MusicPlayerContextProviderProps> = ({
         () =>
             createAppMachine({
                 socket,
+                musicRoomBroadcastChannel: PLATFORM_OS_IS_WEB
+                    ? new BroadcastChannel('music_room_channel')
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Nice! I've never used `BroadcastChannel` 😮